packages/core/helper-plugin/lib/src/utils/getAPIInnerErrors/index.js

```diff
@@ -1,16 +1,30 @@
+function getErrorPayload(error) {
+  const response = error.response.data;
+
+  if (['YupValidationError', 'ValidationError'].includes(response.error.name)) {
+    return response.error.details.errors;
+  }
+
+  return [response.error];
+}
+
 export default function getAPIInnerErrors(error, { getTrad }) {
-  const errorPayload = error.response.data.error.details.errors;
-  const validationErrors = errorPayload.reduce((acc, err) => {
-    acc[err.path.join('.')] = {
-      id: getTrad(`apiError.${err.message}`),
-      defaultMessage: err.message,
-      values: {
-        field: err.path[err.path.length - 1],
-      },
+  return getErrorPayload(error).reduce((acc, err, index) => {
+    const path = err?.path ? err?.path.join('.') : `${err.name}.${index}`;
+    const { message } = err;
+
+    acc[path] = {
+      id: getTrad(`apiError.${message}`),
+      defaultMessage: message,
+
+      // Only errors of type YupValidationError contain a path
+      values: err?.path
```

Comment:
maybe it can be useful to create a variable just for this check to avoid to do it in every place because every time it is transpiled to `err === null || err === void 0 ? void 0 : err.path`

Reply:
Refactored in fae4e10b816f5d2b95011b8348ef63306afe8fce